### PR TITLE
Ny Final Projects

### PR DESCRIPTION
**Pull request by SamFritz**
Addresses [issue #159 ](https://github.com/archivesunleashed/archivesunleashed.org/issues/159)

- [x] Transforms google slides into PDF of final projects
- [x] Adds in final projects to NY Event page following standard formatting (participants names, screenshot image, link to PDF)

tested and looks good on my end. @ianmilligan1 would you mind reviewing to see if there is any missing?

### CODE REVIEW

File: content/new-york/index.md [hunk 1]
```diff
@@ -38,20 +40,61 @@ For more information on AUT and the Cloud, please visit http://archivesunleashed
...
 
+## Team Projects
+
+<table>
+  <tr>
+    <td><b>Team: Latin American and Caribbean Contemporary Art Web Archive</b>
+      <ul>
+        <li>Samantha Abrams, Ivy Plus Libraries Confederation</li>
+        <li>Sumitra Duncan, Frick Art Reference Library/NYARC</li>
+        <li>Mary Nakija, Pratt / NYARC</li>
```
Comment (ianmilligan1) [inline content/new-york/index.md:51]: Check spelling here - I think it is Bakija

File: content/new-york/index.md [hunk 2]
```diff
@@ -14,13 +14,15 @@ Due to the rapidly-evolving COVID-19 pandemic situation, we have decided to hold
 
 ## **About**
 
-We invite archivists, researchers (from the humanities, social sciences, and beyond), librarians, computer scientists, and web archiving enthusiasts to join us over the course of two days to collaboratively work with web collections and explore cutting-edge research tools through hands-on experience.
+The Archives Unleashed Team  partnered with <a href="https://library.columbia.edu">**Columbia University Libraries**</a> to host the fourth Archives Unleashed datathon. This event brought together archivists, researchers (from the humanities, social sciences, and beyond), librarians, computer scientists, and web archiving enthusiasts to collaboratively work with web collections and explore cutting-edge research tools through hands-on experience.
```
Comment (ianmilligan1) [inline content/new-york/index.md:17]: looks like an extra space after `Team` (but it rendered fine in hugo fwiw)

File: content/new-york/index.md [hunk 3]
```diff
@@ -14,13 +14,15 @@ Due to the rapidly-evolving COVID-19 pandemic situation, we have decided to hold
...
-The Archives Unleashed Team has partnered with <a href="https://library.columbia.edu">**Columbia University Libraries**</a> to host the fourth Archives Unleashed datathon.
+## Datathon Materials
 
-This event will bring together a small group to experiment with the newest release of the Archives Unleashed Toolkit and Cloud, and to kick-off collaboratively inspired research projects. Participants will have access to analytics software and specialists, and will be exposed to the process of working with web archive files at scale.
+If you were not able to join us at the datathon, feel free to check out our resources available for use: 
 
-For more information on AUT and the Cloud, please visit http://archivesunleashed.org/.
+* [Video] [Archives Unleashed New York Datathon (Online)](https://youtu.be/Io6RvhqHfe4).
+
+* [PDF Slides] [[Datathon Opening Presentation](/images/AU-NY-IntroSlides-reduced.pdf")]
```
Comment (ianmilligan1) [inline content/new-york/index.md:25]: Maybe remove the square brackets from the Datathon Opening Presentation - or add them on the video link, just so they look consistent. Screenshot attached to show how they're rendering:

Screen Shot 2020-04-03 at 5 25 36 PM

File: content/new-york/index.md (hunk 1 above)
Comment (SamFritz) [reply content/new-york/index.md:51]: thanks for catching!

**Pull request merged**